### ISSUE TITLE
Cannot restore properly if history control module is installed

### ISSUE THREAD

**Issue by iron3oxide** [saltcorn/saltcorn#2680]
For now I can only confirm that this happens on 0.9.3 and 1.0.0-beta.14. I have made a manual backup of a working system on vesion 1.0.0-beta.14. On both tested versions the restore process ends with the following error:
```
TypeError: Cannot read properties of null (reading 'fields')
    at Object.fields (/home/saltcorn/.local/share/saltcorn-plugins/plugins_folder/@saltcorn/history-control/table-provider.js:102:18)
    at State.refresh_tables (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/db/state.ts:482:43)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Table.update (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/table.ts:2340:5)
    at install_pack (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/admin-models/models/pack.ts:493:9)
    at restore (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/admin-models/models/backup.ts:544:3)
    at /home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/server/routes/admin.js:1494:17
```

Some of the backup was successfully restored in both cases, but at minimum the table I had enabled history control on was missing.

**Comment by iron3oxide** [comment]
The same error is thrown when attempting to "Clear all" in the system settings.

**Comment by iron3oxide** [comment]
Can confirm that removing the module and all history tables, then backing up again and restoring works.

**Comment by glutamate** [comment]
fixed in #2704
